README.md

```diff
@@ -1,5 +1,5 @@
 # Fruit shop
-Let's imagine that we have a fruit store. Every day in the store there are a number of activities, 
+Let's imagine that we have a fruitTransaction store. Every day in the store there are a number of activities, 
```

Comment:
Do not change the Readme file. Rollback all changes before re-request new review.

Reply:
Sorry, I don't know how it happened.